docs/main/concepts/architecture.md

```diff
@@ -52,21 +52,20 @@ The implementation pattern used by data engines to provide high availability is
 Using a single controller to implement synchronous replication of data to fixed set of nodes (instead of distribution via multiple metadata controller), reduces the overhead in managing the metadata and also reduces the blast radius related to a node failure and other nodes participating in the rebuild of the failed node. 
 
 The OpenEBS volume services layer exposes the volumes as:
-- Device or Directory paths in case of Local PV, 
-- iSCSI Target in case of cStor and Jiva
-- NVMe Target in case of Mayastor.
+- Device or Directory paths in case of Local Engine
+- NVMe Target in case of Replicated Engine
 
 ### Volume Data Layer 
 
-OpenEBS Data engines create a Volume Replica on top of the Storage Layer. Volume Replicas are pinned to a node and are created on top of the storage layer. The replica can be any of the following:
+OpenEBS Data Engines create a Volume Replica on top of the Storage Layer. Volume Replicas are pinned to a node and are created on top of the Storage Layer. The replica can be any of the following:
```

Comment:
storage layer case-change not needed IMO

Reply:
This has been modified now. :)